libs/product/src/reducers/index.ts

```diff
@@ -131,3 +131,9 @@ export const selectBestSellersIdsState : MemoizedSelector<object, string[]> = cr
   selectBestSellersState,
   fromBestSellers.getBestSellersIds
 );
+
+export const selectBestSellersProducts: MemoizedSelector<object, DaffProduct[]> = createSelector(
```

Comment:
needs a test

Reply:
addressed in d664048f86694c3951420b278205f1f6ca68860e